document/CoreFormat.mdk

```diff
@@ -711,17 +833,14 @@ The `stacks` property is intended for use by static analysis tools that collects
 in the process of producing results.
 
 Each `stack` object in the stacks array shall describe a single call stack.
-A call stack is a sequence of nested function calls simulated by the tool in the course of performing a flow analysis.
+A call stack is a sequence of nested function calls.
 A stack is in turn represented by an array of `annotatedCodeLocation` objects.
 This array shall include every function call in the stack for which the tool has information,
 and the entries that are present shall occur in order with the innermost call first and the outermost call last.
 
-A single result may contain one or more call stacks, one or more execution flows (&sect;[#result-executionFlows]), or both.
-
 ~ IsoNumberedNote { @IsoNumberedNote=1}
 The SARIF file format allows multiple call stacks within a single `result` object
-to allow for the possibility that a static analysis tool might simulate more than one call stack
-in the course of producing a single result.
+to allow for the possibility that a single result might be provoked by way of more than one call stack.
```

Comment:
i don't think a stack 'provokes' a result and this passive voice could be eliminated by something like 'to allow for the possibility that more than one call stack may be relevant to a single result'. or some such.